### ISSUE TITLE
Read until the end of the stdin is not working

### ISSUE THREAD

**Issue by MuhammadSawalhy** [xeluxee/competitest.nvim#11]
I have tried this snippet of code to read until the last line of input:

```cpp
std::string a;
while(std::cin >> a) {
  // do something
}
```

And when I `CompetiTestRun` it resulted in _TIMEOUT_. But it works in the terminal with redirecting a file to the stdin of the program.

```bash
./main < input.txt
```

**Comment by xeluxee** [comment]
Please provide `main.cpp` and `input.txt`

**Comment by MuhammadSawalhy** [comment]
```cpp
#include <bits/stdc++.h>

int32_t main() {
  ios_base::sync_with_stdio(false);
  cin.tie(NULL);
  string a, b;

  while (cin >> a) {
    cin >> b;
    std::cout << a << " * " << b << std::endl;
  }

  return 0;
}
```

```
12
12
2
222222222222222222222222
```

**Comment by xeluxee** [comment]
Fixed with aa3bbfbd4432ef28f026148c99961811562c77ef
